kubernetes/skipper-server/create.sh

```diff
@@ -14,7 +14,72 @@ function kubectl_create() {
   return 0
 }
 
+
+function generate_manifest() {
+cat << EOF > ./skipper.yml
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: skipper
+spec:
+  replicas: 1
+  selector:
+    name: skipper
+  template:
+    metadata:
+      labels:
+        name: skipper
+    spec:
+      containers:
+      - name: skipper
+        image: springcloud/spring-cloud-skipper-server:$SKIPPER_VERSION
```

Comment:
i think the same would apply with the comment in `kubernetes/server/create.sh`

Reply:
Indeed there are no `latest` in the skipper (perhaps @markpollack knows why?) but it fits well current acceptance tests model :)